### DISCUSSION TITLE
Standalone Wi-Fi image streaming with AI deck

### DISCUSSION THREAD

**Discussion by szgezal** [bitcraze/discussions#1497] [Q&A]
Hi,
Is it possible to use the AI deck as a standalone device and stream images from the Himax camera?

I am experiencing the same problem discussed in these issues:
https://github.com/bitcraze/crazyflie-firmware/issues/1205
https://github.com/bitcraze/aideck-gap8-examples/issues/106
The Wi-Fi streaming example works as intended, but when I try to fly the drone (with the Crazyswarm2 project), the AI deck network cuts off, and I have to restart the drone to get the streaming working again. Is there any solution for this problem?

Therefore, I would like to use the AI deck as a standalone board. However, when I power the deck on with VCOM, it does not seem to function as an access point. Previously, I set up the board with menuconfig and if the deck is attached to the drone (Crazyflie 2.1 or Bolt), the Wi-Fi works. Do I need to flash anything to the NINA module using JTAG, or is there another step I might have missed?

Thanks in advance!

**Comment by knmcguire** [comment]
Hi! 

I'm guessing that the motors drain the power to the aideck and it resets, so you can do several things with that:
- Have the motors run before take off 
- Have it take off more slowly
- Use the [thrust upgrade kit](https://store.bitcraze.io/collections/bundles/products/thrust-upgrade-bundle-for-crazyflie-2-x) with [the better battery](https://store.bitcraze.io/products/350mah-lipo-battery).

If you upload the wifi example of the getting started tutorial that should work stand alone as an access point without the communication of the crazyflie: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/. However we don't give guarantees on how well it works without the crazyflie so some self debugging is required.
